src/main/resources/assets/app/scripts/templates/job_detail_view.hbs

```diff
@@ -19,6 +19,21 @@
       </div>
       <span class="help-inline hide"></span>
     </div>
+    
+     <div class="row-fluid control-group job-detail-row job-detail"
+         data-toggle="tooltip"
+         data-html="true"
+         data-title="Description of the job."
+         data-placement="bottom">
+      <div class="detail">Description</div>
```

Comment:
NIT: indent one more space.  the rest of the file indents in 2 spaces, so let's keep it consistent.  The same applies to the rest of the div block.